public/kibana-integrations/kibana-vis.js

```diff
@@ -205,14 +246,21 @@ class KibanaVis extends Component {
         this.visID === "Wazuh-App-Overview-General-Agents-status";
       const timeFilterSeconds = this.calculateTimeFilterSeconds(
         timefilter.getTime()
-      );
+      );        
+
       const timeRange =
         isAgentStatus && timeFilterSeconds < 900
           ? { from: "now-15m", to: "now", mode: "quick" }
           : timefilter.getTime();
       const filters = isAgentStatus ? [] : discoverList[1] || [];
       const query = !isAgentStatus ? discoverList[0] : {};
-
+      const rawVis = raw ? raw.filter((item) => item && item.id === this.visID) : [];
+      let vizPattern;
+      try {
+        vizPattern = JSON.parse(rawVis[0].attributes.kibanaSavedObjectMeta.searchSourceJSON).index;
```

Comment:
Same here, in this case, we have hidden the errors. Maybe we can log this kind of error, it's not necessary to show a toast.